release/src/version_update/main.py

```diff
@@ -111,8 +111,8 @@ def updatePropertiesFile(data, modules, currentVersions):
         latestVersion = module['version']
 
         if moduleName == 'java.arrays':
-            version = compareVersion(latestVersion, currentVersions['stdlibJarraysVersion'])
-            line = "stdlibJarraysVersion=" + version + "\n"
+            version = compareVersion(latestVersion, currentVersions['stdlibJavaArraysVersion'])
```

Comment:
Since we are using the same string in different places, it's better if we use a constant for this string value. That way, it will be easier to make changes to these.

Reply:
Updated